event_webhook_types_test.go

```diff
@@ -185,24 +184,24 @@ func TestMergeEventUnmarshal(t *testing.T) {
 	if event.ObjectAttributes.LastCommit.ID != "da1560886d4f094c3e6c9ef40349f7d38b5d27d7" {
 		t.Errorf("ObjectAttributes.LastCommit.ID is %v, want %s", event.ObjectAttributes.LastCommit.ID, "da1560886d4f094c3e6c9ef40349f7d38b5d27d7")
 	}
-	if event.ObjectAttributes.Assignee.Name != expectedName {
-		t.Errorf("Assignee.Name is %v, want %v", event.ObjectAttributes.Assignee.Name, expectedName)
+	if event.Assignees[0].Name != expectedName {
+		t.Errorf("Assignee.Name is %v, want %v", event.Assignees[0].Name, expectedName)
 	}
 
-	if event.ObjectAttributes.Assignee.Username != expectedUsername {
-		t.Errorf("ObjectAttributes is %v, want %v", event.ObjectAttributes.Assignee.Username, expectedUsername)
+	if event.Assignees[0].Username != expectedUsername {
+		t.Errorf("ObjectAttributes is %v, want %v", event.Assignees[0].Username, expectedUsername)
 	}
 
-	if event.User.ID != 42 {
-		t.Errorf("User ID is %d, want %d", event.User.ID, 42)
+	if event.User.ID != expectedID {
+		t.Errorf("User ID is %d, want %d", event.User.ID, expectedID)
 	}
 
-	if event.User.Name != expectedName {
-		t.Errorf("Username is %s, want %s", event.User.Name, expectedName)
+	if event.User.Name != "Administrator" {
```

Comment:
Please use the constant (and make sure that the test input uses the same values)...